github/repos.go

```diff
@@ -985,13 +985,13 @@ func (s *RepositoriesService) RemoveAdminEnforcement(ctx context.Context, owner,
 
 // Topics represents a collection of repository topics.
 type Topics struct {
-	Names []string `json:"names,omitempty"`
+	Names []string `json:"names"`
 }
```

Comment:
Since this type is replaced in favor of `[]string` in the exported method signatures, it's no longer used, and should be either made unexported, or removed (replaced by ad-hoc type definitions in the methods that need it).